app/assets/stylesheets/blogs.scss

```diff
@@ -0,0 +1,37 @@
+.blogs {
+  .blog-header {
+    @include clearfix;
+
+    .icon {
+      float: left;
+      width: 64px;
+      height: 64px;
+    }
+
+    .title {
+      float: left;
+      margin-left: 20px;
+      line-height: 60px;
+    }
+  }
+
+  .blog-colophon {
+    margin-bottom: 10px;
```

Comment:
Properties should be ordered background-color, border-radius, font-size, margin-bottom, padding